shared/base/src/main/java/com/ivy/base/Constants.kt

```diff
@@ -0,0 +1,5 @@
+package com.ivy.base
+
+object Constants {
+    const val TRANSFER_HEADER_GRADIENT_THRESHOLD = 0.35f
```

Comment:
No need for this constant file. Just put the constant as `private val constant` in the file where you're using it